completion.go

```diff
@@ -105,12 +141,12 @@ func (c *Client) CreateCompletion(
 		return
 	}
 
-	if request.Model == GPT3Dot5Turbo0301 || request.Model == GPT3Dot5Turbo {
-		err = ErrCompletionUnsupportedModel
+	urlSuffix := "/completions"
+	err = checkEndpointSupportsModel(urlSuffix, request.Model)
```

Comment:
we can make `checkEndpointSupportsModel` return a boolean flag, and use the previously defined errors e.g. `ErrChatCompletionInvalidModel`